### PR TITLE
Integrated TOIF into apache-fineract

### PR DESCRIPTION
**Pull request by ThisuraThejith**
* Reports are generated in appache-fineract/reports/toif
* Task is not executed upon build, due to time restriction
* ./gradlew toif will run the scanning task

### CODE REVIEW

File: fineract-provider/toif/housekeeping/housekeeping.txt [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+#		Entities
+###########################
+
+SegmentDescription=Segment.
+
+#projectId=name;description
+project1=apache-fineract;Apache Fineract:A Platform for Microfinance.
+
+#personId=name;email;phone
+person1=mifos-developer;dev@fineract.apache.org;555-1234
```
Comment (nazeer1100126) [inline fineract-provider/toif/housekeeping/housekeeping.txt:26]: Suggest not to have mifos names as this is Apache product now.

Comment (ThisuraThejith) [reply fineract-provider/toif/housekeeping/housekeeping.txt:26]: @nazeer1100126 will do it.

Comment (avikganguly01) [reply fineract-provider/toif/housekeeping/housekeeping.txt:26]: @ThisuraThejith Can you update your PR with the suggested change so it can be merged?

**Comment by vorburger** [comment]
@ThisuraThejith thank you very much for your interest in Fineract/Mifos!  I'm an (old) volunteer who got reminded of Mifos/Fineract this week-end at the FOSDEM conference, and I'm taking a moment to help the project clean up some of it's old issues and pull requests etc.  

We were wondering whether you are still alive, and would be willing to rebase this PR and take any previous review feedback into account?  If we don't hear from you within 2 weeks, we will take the liberty to close this PR, to avoid confusion for future new contributors looking at this repository - hope you understand and that's fair.

/Cc FYI @apache/fineract-committers

**Comment by vorburger** [comment]
Friendly service reminder: I will close this PR in 3 days to help this project have a clean list of active PRs.

**Pull request closed without merging**